include/FanFicArchive.py

```diff
@@ -0,0 +1,85 @@
+from bs4 import BeautifulSoup; from urllib import request; from re import match; import requests
+
+parser = "lxml"
+
+def fanfic_req(link): return BeautifulSoup(request.urlopen(link).read(), parser)
```

Comment:
This function name is unclear. It should also probably be an internal member of a class, rather than something semi-public that exposes implementation details like the BeautifulSoup object.

Reply:
It is a common-use functions for multiple websites

Reply:
In this one, my suggestion would be to mark the method with an underscore (`_fanfic_req`) to inform other readers that it is not intended to be used by other modules. Though it might still want to be squirrelled away into a class.